docs/integrations/sources/safetyculture.md

```diff
@@ -0,0 +1,57 @@
+# SafetyCulture
+
+This is the setup guide for the Safetyculture source connector which ingests data from the Safetyculture API.
+
+## Prerequisites
+
+This source uses the Authorization Bearer Token for handling requests. In order to obtain the credientials, you must first create a Safetyculture account.
+The API usage is only availabe for paid plans https://www.Safetyculture.com/
+
+You can create an API token under Account Settings -> Integrations -> API Tokens
+You can find more about their API here https://www.Safetyculture.com/api-reference/
```

Comment:
Have you tested this link? Because it 404s for me, and I see their docs here instead. https://developer.safetyculture.com/reference/organisationservice_getusergroups 

Whichever, make all links actual markdown links people can click on. /cc @marcosmarxm

Reply:
Fixed the broken link